task6.js

```diff
@@ -0,0 +1,13 @@
+// Write a function checkEvenOdd(n) that uses a ternary operator to return 'Even' if the number is even and 'Odd' if it's odd.
+// Sample output:
+// checkEvenOdd(5) => Odd
+
+const checkEvenOdd = (n) => {
+  if (typeof n !== "number" || Number.isNaN(n) || !Number.isInteger(n)) {
```

Comment:
You can combine the checks for number type and NaN into a single condition
` if (typeof n !== "number" || !Number.isInteger(n)) { ... }`